parity-util-mem/Cargo.toml

```diff
@@ -27,8 +27,6 @@ optional = true
 [features]
 default = ["std", "ethereum-impls"]
 std = []
-# when activated mem is removed through volatile primitive instead of clear_on_drop crate
```

Comment:
```
elastic-array = { version = "*", optional = true }
ethereum-types = { version = "*", optional = true }
parking_lot = { version = "*", optional = true }
```
Can you add in the versions used please?

Reply:
Actually this is a case where I have some use for "*" : to avoid having to change those version when using util-mem (for instance project A uses it and uses api from a version of eth-type, project B another version).
The implementation is kind of lose in some way (can break but it is not very likely). This is far from ideal and may require tweaking cargo lock when multiple version of the libs are in use (it happened to me).
To avoid using it the only way I see is to implement the trait in the crates, but that is something I wanted to avoid (or implement the trait on multiple version which may be needed at some point (I think it can be a bit tricky)).
I remember there was some api change on our crates that would requires substantial work to use them in ethereum at the time, maybe it is not the case anymore and the implementation can move back there (not parking_lot obviously).

Reply:
I see what you’re saying, but we can’t publish this crate with “*”. :/